libs/langgraph-checkpoint-mongodb/langgraph/checkpoint/mongodb/aio.py

```diff
@@ -94,16 +113,23 @@ async def from_conn_string(
         writes_collection_name: str = "checkpoint_writes_aio",
         **kwargs: Any,
     ) -> AsyncIterator["AsyncMongoDBSaver"]:
+        """Create asynchronous checkpointer
+
+        This includes creation of collections and indexes if they don't exist
+        """
         client: Optional[AsyncIOMotorClient] = None
         try:
             client = AsyncIOMotorClient(conn_string)
-            yield AsyncMongoDBSaver(
+            saver = AsyncMongoDBSaver(
                 client,
                 db_name,
                 checkpoint_collection_name,
                 writes_collection_name,
                 **kwargs,
             )
+            await saver.setup()
```

Comment:
(nit) we usually recommend that users manually call `.setup()` on their checkpointers the first time

Reply:
My preference is to allow users to use from_conn_string to automatically set up collections and indexes. Vadym and I discussed, and he doesn't have objection. One is free to set up collections and indexes manually, and/or call the constructor and setup separately. I'm off from this afternoon. Would love an approval on this.